server/routes/v1/deployment.ts

```diff
@@ -111,3 +113,28 @@ export const postDeployment = [
     })
   },
 ]
+
+export const resetDeploymentApprovals = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    const user = req.user
+    const body = req.body as any
+    const deployment: Deployment = await DeploymentModel.findOne({ uuid: body.uuid })
```

Comment:
You _cannot_ cast to `Deployment` here, as it's a `Deployment | null` type.  You _must_ check that Deployment is a thing, or this function will error when it's not found.

Reply:
Resolved